packages/fether-electron/src/main/app/options/index.js

```diff
@@ -0,0 +1,42 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+import extend from 'extend';
+
+import { DEFAULT_OPTIONS, TASKBAR_OPTIONS } from './config';
+
+let hasCalledInitFetherAppOptions = false;
+
+class FetherAppOptions {
+  options = {};
+
+  create = (withTaskbar, customOptions) => {
+    if (hasCalledInitFetherAppOptions) {
+      throw new Error('Unable to initialise Fether app options more than once');
+    }
+
+    // Allow user to get/set options prior or to pass custom options
+    this.options = withTaskbar
+      ? extend(
```

Comment:
Consider using either `this.options = {...this.options, DEFAULT_OPTIONS, TASKBAR_OPTIONS, customOptions}` or `Object.assign(this.options, DEFAULT_OPTIONS, TASKBAR_OPTIONS, customOptions)` (so that we can remove one dependency)